src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/GeneratedDocumentPublisher.cs

```diff
@@ -8,8 +8,8 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     internal abstract class GeneratedDocumentPublisher : ProjectSnapshotChangeTrigger
     {
-        public abstract void PublishCSharp(string filePath, SourceText sourceText, long hostDocumentVersion);
+        public abstract void PublishCSharp(string filePath, SourceText sourceText, int? hostDocumentVersion);
```

Comment:
If we're publishing C# we should always have a host document version shouldn't we? Ditto with HTML